jzy3d-jGL/jzy3d-jGL-awt/src/main/java/jgl/wt/awt/listener/GLUTReflectiveCallbackListener.java

```diff
@@ -0,0 +1,132 @@
+package jgl.wt.awt.listener;
+
+import java.awt.*;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Old reflection based 'Method' callback based on method name.
+ */
+public class GLUTReflectiveCallbackListener implements GLUTListener {
+
+  private Method reshapeMethod;
```

Comment:
Despite not a big deal here, I tend to always set fields protected in Jzy3D to ease overriding.